src/benchmarklib/benchmark_runner.cpp

```diff
@@ -305,21 +305,27 @@ void BenchmarkRunner::_create_report(std::ostream& stream) const {
                      return static_cast<double>(std::chrono::duration_cast<std::chrono::nanoseconds>(duration).count());
                    });
 
-    nlohmann::json benchmark{
-        {"name", name},
-        {"iterations", query_result.num_iterations.load()},
-        {"iteration_durations", iteration_durations},
-        {"avg_real_time_per_iteration", time_per_query},
-        {"items_per_second", items_per_second},
-        {"time_unit", "ns"},
-    };
+    nlohmann::json benchmark{{"name", name},
+                             {"iterations", query_result.num_iterations.load()},
+                             {"iteration_durations", iteration_durations},
+                             {"avg_real_time_per_iteration", time_per_query},
+                             {"items_per_second", items_per_second},
+                             {"time_unit", "ns"}};
 
     benchmarks.push_back(benchmark);
   }
 
+  // Gather information on the (estimated) table size
+  auto footprint = 0ull;
+  for (const auto& table_pair : StorageManager::get().tables()) {
+    footprint += table_pair.second->estimate_memory_usage();
+  }
+
   const auto total_run_duration_seconds = std::chrono::duration_cast<std::chrono::seconds>(_total_run_duration).count();
-  nlohmann::json report{
-      {"context", _context}, {"benchmarks", benchmarks}, {"total_run_duration (s)", total_run_duration_seconds}};
+
+  nlohmann::json summary{{"footprint", footprint}, {"total_run_duration (s)", total_run_duration_seconds}};
```

Comment:
`memory_consumption`? Footprint is too vague imo.

Reply:
Memory consumption could also include the memory consumed by operators. :/

Reply:
Or even better `memory_consumption_in_bytes` or something like that to denote the unit. These number will be huge for TPCH sf1.0 and larger scales.

Reply:
Would you say footprint does not include the operators? I would assume that it does as well.

Reply:
`table_size_in_bytes`?

Reply:
Table size sounds less ambiguous.